tests/models/test_aurora.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Don't review this yet because this entire script needs to be revamped to load dummy test data as xarray.